src/user-guide/running-workflows/workflow-run-modes.rst

```diff
@@ -0,0 +1,114 @@
+.. _workflow-run-modes:
+
+Workflow Run Modes
+------------------
+
+Cylc can run a workflow without running the real jobs, which may be
+useful when developing workflows. Tasks 
+in :ref:`workflow-run-modes.dummy` and
+:ref:`workflow-run-modes.simulation` can help you to understand how different outcomes
+and run lengths will affect the workflow.
+
+To start a workflow in one of these modes:
+
+.. code-block:: console
+
+   $ cylc play --mode=dummy <workflow-id>
+   $ cylc play --mode=simulation <workflow-id>
+
+.. admonition:: Limitations
+
+   * Workflow run mode is recorded in the workflow run database. Cylc will not let you
+     *restart* a dummy mode workflow in live mode, or vice versa. Instead,
+     install a new instance of the workflow and run it from scratch in the new mode.
+   * Workflows run in Dummy and Simulation mode do not allow the run mode of
+     individual tasks to be overridden by :ref:`task-run-modes.skip`
+
+
+.. _workflow-run-modes.dummy:
+
+Dummy Mode
+----------
+
+**Dummy mode** replaces real jobs with background jobs on the
+scheduler host which use ``sleep`` to simulate the run length according to
+the settings described for simulation mode.
+This avoids :term:`job runner` directives that request compute
+resources for real workflow tasks, and it allows any workflow configuration to
+be run locally in dummy mode.
+
+.. admonition:: Limitations
+
+   * Dummy mode can only be applied to a whole workflow.
+   * Dummy tasks run locally, so dummy mode does not test communication with remote
+     job platforms. However, it is easy to write a live-mode test workflow with
+     simple ``sleep 10`` tasks that submit to a remote platform.
+
+
+.. _workflow-run-modes.simulation:
+
+Simulation Mode
+---------------
+
+**Simulation mode** does not run real jobs, and does not generate job
+log files.  Instead, the scheduler internally simulates task completion to evolve
+the workflow.
+
+
+Simulated Run Length
+^^^^^^^^^^^^^^^^^^^^
+
+The default dummy or simulated job run length is 10 seconds. It can be
+changed with :cylc:conf:`[runtime][<namespace>][simulation]default run length`.
+
+If :cylc:conf:`[runtime][<namespace>]execution time limit` and
+:cylc:conf:`[runtime][<namespace>][simulation]speedup factor` are both set,
+run length is computed by dividing the
+:cylc:conf:`[runtime][<namespace>]execution time limit` by
+:cylc:conf:`[runtime][<namespace>][simulation]` speedup factor.
+
+Simulated Failure
+^^^^^^^^^^^^^^^^^
+
+Tasks always complete all custom outputs, and by default they will succeed.
+
+If you want to test individual workflow pathways, use
+:ref:`skip mode <task-run-modes.skip>`.
```

Comment:
Sorry, what does this mean?

Reply:
If you have a case:

```
a:foo? => f
b:bar? => b
```

Simulation mode will always run f & b, but if you want to simulate only emitting one of them then your only option at present is skip mode.

How about

```suggestion
If you want to test emitting only some custom outputs
you can run the workflow in live mode with task run modes
set to :ref:`skip mode <task-run-modes.skip>`.
```